codeserver/ubi9-python-3.12/Dockerfile.cpu

```diff
@@ -171,16 +172,23 @@ ENV NGINX_VERSION=1.24 \
 
 # Modules does not exist
 RUN dnf install -y https://download.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
-    INSTALL_PKGS="bind-utils nginx nginx-mod-stream nginx-mod-http-perl fcgiwrap initscripts chkconfig supervisor" && \
+    INSTALL_PKGS="bind-utils nginx nginx-mod-stream nginx-mod-http-perl httpd systemd" && \
```

Comment:
I could not find where systemd is actually being used

Reply:
right, I removed it, it was there from the original intent to replace the supervisord